src/screens/AddNewItemScreen/index.tsx

```diff
@@ -34,14 +39,49 @@ export const AddNewItemScreen: FC = () => {
     navigation.goBack();
   };
 
+  const onConfirmDate = (date: Date) => {
+    setOpen(false);
+    setDate(date);
+  };
+  const onCancelDate = () => {
```

Comment:
where is enter line?
I am tired of paying your attention on it in every request!!!